src/Pipl.php

```diff
@@ -13,42 +13,34 @@
  */
 class Pipl
 {
-
-    /**
-     * @var ClientInterface
-     */
```

Comment:
Why did you removed the code block here ?

Reply:
You are already type hinting in a constructor what kind of variable is it (see in the search method, PHPStorm knows where to follow `->get()`). 
Even with the block included, it does nothing because your format is wrong. The doc block is missing the variable name. Correct format is `@var $variable Class`.

Reply:
I believe it's $variable_name is optional  in docBlock
But feel free to add it in the docBlock if you want

Reply:
I wouldn't personally add it because of the type hint. If you require it I will revert this one.